definitions/subworkflows/single_sample_sv_callers.cwl

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Subworkflow to allow calling different SV callers which require bam files as inputs"
+
+requirements:
+  - class: InlineJavascriptRequirement
+
+inputs:
+    cram:
+        type: File
+    reference:
+        type: string
+
+    cnvkit_access:
+        type: File?
+    cnvkit_bait_intervals:
+        type: File?
+    cnvkit_diagram:
+        type: boolean?
+    cnvkit_drop_low_coverage:
+        type: boolean?
+    cnvkit_method:
+        type: string?
+    cnvkit_reference_cnn:
+        type: File?
+    cnvkit_scatter_plot:
+        type: boolean?
+
+    manta_call_regions:
+        type: File?
+    manta_non_wgs:
+        type: boolean?
+    manta_output_contigs:
+        type: boolean?
+
+    smoove_exclude_regions:
+        type: File?
+
+outputs:
+    intervals_antitarget:
+        type: File?
+        outputSource: run_cnvkit/intervals_antitarget
+    intervals_target:
+        type: File?
+        outputSource: run_cnvkit/intervals_target
+    normal_antitarget_coverage:
+        type: File?
+        outputSource: run_cnvkit/normal_antitarget_coverage
+    normal_target_coverage:
+        type: File?
+        outputSource: run_cnvkit/normal_target_coverage
+    reference_coverage:
+        type: File?
+        outputSource: run_cnvkit/reference_coverage
+    cn_diagram:
+        type: File?
+        outputSource: run_cnvkit/cn_diagram
+    cn_scatter_plot:
+        type: File?
+        outputSource: run_cnvkit/cn_scatter_plot
+    tumor_antitarget_coverage:
+        type: File
+        outputSource: run_cnvkit/tumor_antitarget_coverage
+    tumor_target_coverage:
+        type: File
+        outputSource: run_cnvkit/tumor_target_coverage
+    tumor_bin_level_ratios:
+        type: File
+        outputSource: run_cnvkit/tumor_bin_level_ratios
+    tumor_segmented_ratios:
+        type: File
+        outputSource: run_cnvkit/tumor_segmented_ratios
+    manta_diploid_variants:
+        type: File?
+        outputSource: run_manta/diploid_variants
+    manta_somatic_variants:
+        type: File?
+        outputSource: run_manta/somatic_variants
+    manta_all_candidates:
+        type: File
+        outputSource: run_manta/all_candidates
+    manta_small_candidates:
+        type: File
+        outputSource: run_manta/small_candidates
+    manta_tumor_only_variants:
+        type: File?
+        outputSource: run_manta/tumor_only_variants
+    smoove_output_variant:
```

Comment:
Should this be called `smoove_output_variants` to match the other `*_variants` outputs?

Reply:
good catch. Just updated the output name